Fuselag Notam (Independent Publisher)

SalesLighter opened the pull request:
Description: This will retrieve NOTAM (Notice to airmen) data from the FAA using a valid API key. 

See further instructions in the ReadMe file.

Options:

I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

SalesLighter committed:
FAA Notam

SalesLighter committed:
initial changes : falana kidd

SalesLighter committed:
Properties file and readme file

SalesLighter committed:
removing test file

vmanoharas reviewed (changes requested):
Hello @SalesLighter, 

If your connector is an independent publisher, please refer the public documentation here. https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission-ip

Also please resolve below comments. 

Please agreed the CLA
Please go through the template and add required files and sections - https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/Independent%20Publisher

SalesLighter commented:
@vmanoharas Why are we submitting a CLA license agreement for a connector self created?

SalesLighter committed:
file renamed - fk

SalesLighter committed:
@microsoft-github-policy-service agree company="Double Axis"

SalesLighter committed:
Independent Publisher Icon

.ico file

SalesLighter committed:
@microsoft-github-policy-service agree company="Double Axis"

SalesLighter committed:
-added additional properties

SalesLighter committed:
@microsoft-github-policy-service agree company="Double Axis"

SalesLighter committed:
@microsoft-github-policy-service agree company="Double Axis"

SalesLighter committed:
@microsoft-github-policy-service agree company="Double Axis"

SalesLighter committed:
@microsoft-github-policy-service agree

SalesLighter commented:
Issues corrected 
summary property and contact /info property
license/cla does not work as expected

SalesLighter commented:
@vmanoharas , the modifications were made to the connector. However, there are issues with the cla/license agreement. I constantly received a message stating that the license is not agreed upon. I have tried submitting the following parameters: 
**@microsoft-github-policy-service agree** and
**@microsoft-github-policy-service agree company="Double Axis"**
What are these parameters attempting to sync or map to? Our profile name, etc?

SalesLighter commented:
@microsoft-github-policy-service agree company="Microsoft"

SalesLighter commented:
@vmanoharas , signed the cla and all checks have passed. Please advise on next steps.

vmanoharas commented on independent-publisher-connectors/FAANotam/README.md:
```diff
@@ -0,0 +1,13 @@
+# FAANotamCRM
```
Hi @SalesLighter, 
Please update readme.md file as per the template in the link.  Please make sure that you added all the required fields. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

vmanoharas commented on independent-publisher-connectors/FAANotam/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,300 @@
+    "title": "FuseLagNotam1.1",
+    "description": "This connector will allow users to retrieve NOTAMs (Notices to airmen) directly from the FAA. Notams are used to notify pilots of any current and future issues within the United States Airspace.",
+    "contact":{
+      "name":"Falana Kidd",
```
Please add all the contact fields. 
  "contact": {
      "name": "Full name",
      "url": "Contact URL",
      "email": "Contact email address"

SalesLighter replied:
Added contact information and modified the Readme file

vmanoharas replied:
Hi @SalesLighter, 

Thank you very much for updating the changes requested. 

Can you please update the title of the PR as mentioned in the checklist, and mentioned as below. 
I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)

Also please go through all the checklist above and check the checkbox.

vmanoharas reviewed (commented):
Please resolve review comments and remove icon files, as this is not needed to add as part of the artifacts, we will take while certifying your connector. 

Additionally, please update the title of the PR from Proposal - FAA Notam - Double Axis to <Connector Name> (Independent Publisher)

SalesLighter committed:
-Added contact information

SalesLighter committed:
-modified Readme.md file

SalesLighter commented:
Renamed PR from: Proposal - FAA Notam - Double Axis to Fuselag Notam (Independent Publisher)

SalesLighter committed:
-Added required screenshots to Readme file demonstrating the use of the feature.

SalesLighter commented:
- Modified the PR title
- Checked the boxes on the PR form
- Modified Readme file and added screenshots demonstrating use of the feature
- Showed successful test inside of the readme file

SalesLighter commented:
@vmanoharas  Please review the requested modifications. All checks have passed.

vmanoharas reviewed (commented):
Hi @SalesLighter, 

thank you very much for resolving my previous review comments.  Please remove all the images in the artifact, they are not required to upload as part of GitHub repository. We will take care of the icon during certification process.

SalesLighter committed:
-Per request: Removed images from ReadME.md and the directory

SalesLighter committed:
-Removed images

SalesLighter committed:
-removed images

SalesLighter commented:
@vmanoharas , Removed the images from the ReadMe file and from the directory

SalesLighter commented:
@vmanoharas , Please review

vmanoharas commented:
Hi @SalesLighter, 

Thank you very much, please remove independent-publisher-icon as well.  We will add it during certification.

SalesLighter committed:
-Removed the icon from the directory

SalesLighter commented:
@vmanoharas - Removed the independent publisher icon from the directory per your request. Please review.

vmanoharas reviewed (approved):
Hello @SalesLighter, 

I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

vmanoharas commented:
[[certify-connector]]

SalesLighter commented:
Hello, I see that the certification is still in progress. Any updates on a certification date?

vmanoharas commented:
Hello @SalesLighter, 

Your connector is now deployed to the Asia region. It will be deployed to all the regions on or before 22nd December.

vmanoharas commented:
Hello Partner,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.
